src/renderer/components/App.tsx

```diff
@@ -54,7 +57,11 @@ export default class App extends React.Component<any, undefined> {
             dialogActions.open(
                 DialogType.FileImport,
                 {
-                    files: acceptedFiles.map((file) => {
+                    files: acceptedFiles.filter((file) => {
+                            return (path.extname(file.path) === ".epub" ||
+                            path.extname(file.path) === ".lcpl");
```

Comment:
Once last nit-picking comment :)

`path.extname(file.path)` is evaluated twice at each iteration. Instead, just store the string value in a constant variable scoped to the function body:

```javascript
files: acceptedFiles.filter((file) => {
    const ext = path.extname(file.path);
    return (ext === ".epub" || ext === ".lcpl");
}
```